web/app/shared/services/scalar/scalar-widget.api.ts

```diff
@@ -43,6 +43,7 @@ export class ScalarWidgetApi {
             data: {
                 description: sticker.description,
                 content: {
+                    body: sticker.description,
```

Comment:
This needs a comment to say it's non-standard, otherwise I'd probably remove it next time I'm in the area.